edflow/custom_logging.py

```diff
@@ -423,21 +423,63 @@ def _create_logger(name, out_dir, level=logging.INFO):
         logger = logging.getLogger(name)
         logger.setLevel(level)
 
+        logger.copyline = lambda msg: logger.log(22, msg)
```

Comment:
do we want to use something specific like `copyline` or something more generic like `colored_info`?
Why is `levelno` 22 used? anything special about that?

Reply:
> do we want to use something specific like copyline or something more generic like colored_info?

Agreed!

> Why is levelno 22 used? anything special about that?

During formatting we can access the level and use this info to switch to color mode.